.github/workflows/cmd.yml

```diff
@@ -59,14 +68,11 @@ jobs:
         id: coursier-cache
         uses: coursier/cache-action@v6.3
 
-      - name: Saving cache-hit-coursier env variable
-        run: echo 'CACHE_HIT_COURSIER=${{ steps.coursier-cache.outputs.cache-hit-coursier }}' >> $GITHUB_ENV
-
       - name: Custom Cache
         uses: actions/cache@v2
         if: ${{ inputs.cache-key != '' && inputs.cache-path != '' }}
         with:
-          key: ${{ inputs.cache-key }}
+          key: ${{ format(inputs.cache-key, matrix.java) }}
```

Comment:
This substitution is needed for https://github.com/playframework/playframework/pull/11259/files#diff-063ca77e012f959eba648db60e6868875fd1e70e5f5ac081193d848f8d61ef32R90:
`play-published-local-jdk{0}-...`
So if the key contains `{0}` it will be replaced with the java version. IMHO that is ok, because the java input always exists. If the key does not contain `{0}` it does not matter, `format` will just return the original string.